<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="BattleWindow">
   <widget class="QPushButton" name="escapeButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Escape</string>
    </property>
   </widget>
   <widget class="QLabel" name="opponentLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 140);
color: black;
font: 14pt &quot;MS UI Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Pokemon2</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>551</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 140);
color: black;
font: 14pt &quot;MS UI Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>640</width>
      <height>460</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;C:/Users/PaulNOWAK/Desktop/Algosup/Moonshot Project/Kritical-Hit/Prototypes/Simulation_Test1/Images/BattleBackground.jpeg&quot;) 100% stretch;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>82</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 140);
color: black;
font: 14pt &quot;MS UI Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Pokemon1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attackButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>playerLabel</zorder>
   <zorder>attackButton</zorder>
   <zorder>opponentLabel</zorder>
   <zorder>statusLabel</zorder>
   <zorder>escapeButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
